docs/redisai.md

```diff
@@ -0,0 +1,460 @@
+# RedisAI Integration
+
+[RedisAI](https://oss.redis.com/redisai/) is a Redis module for executing Deep Learning/Machine Learning models and managing their data. Its purpose is being a "workhorse" for model serving, by providing out-of-the-box support for popular DL/ML frameworks and unparalleled performance.
+RedisGears has built-in integration with RedisAI via a Python plugin that enables the registration of AI flows, and triggering it upon events.
+
+## Setup
+To use RedisAI functionality with RedisGears, RedisAI module should be loaded to the Redis server along with RedisGears.
+The quickest way to try RedisGears with RedisAI is by launching the [`redismod`](https://hub.docker.com/r/redislabs/redismod) Docker container image that bundles together the latest stable releases of Redis and select Redis modules from Redis:
+
+```docker run -p 6379:6379 redislabs/redismod:latest```
+
+Alternatively, you can build RedisAI from its source code by following the instruction [here](https://oss.redis.com/redisai/quickstart/).
+Then, you can run the following command to load the two modules (from RedisGears root directory):
+
+```redis-server --loadmodule ./redisgears.so Plugin ./gears_python.so --loadmodule <path/to/RedisAI-repo>/install-cpu/redisai.so```
+
+## Usage
+
+The integration with RedisAI is enabled via RedisGears' embedded interpreter [Python plugin](runtime.md).
+Use `import redisAI` to import RedisAI functionality to the runtime interpreter.
+
+### Objects
+
+The `redisAI` module contains Pythonic wrappers of RedisAI objects (further explanations about these objects is available in RedisAI [docs](https://oss.redis.com/redisai/master/)):
+* PyTensor - represents a tensor, which is an n-dimensional array of values
+* PyModelRunner - represents a context of model's execution. A model in RedisAI is a computation graph by one of the supported DL/ML framework backends
```

Comment:
* PyModelRunner - represents a context of model's execution. A model is a computation graph for a supported DL/ML framework backend